manifests/windows/install.pp

```diff
@@ -10,6 +10,7 @@
   $install_dir           = undef,
   $install_options       = [],
   $msi_move_locked_files = $::puppet_agent::msi_move_locked_files,
+  $service_names         = $::puppet_agent::params::service_names,
```

Comment:
May be worth adding a comment about why we're not using the `service_names` parameter supplied by the user?